website/content/docs/security/acl/auth-methods/jwt.mdx

```diff
@@ -172,3 +172,12 @@ If multiple methods are needed, another auth method of this type may be created
 with a different name.
 
 @include 'jwt_claim_mapping_details.mdx'
+
+## Token Locality
+
+By default, tokens created by the JWT Auth Method will have local scope and will
+therefore only be valid for the datacenter in which they were created.
+
+In the primary DC, the Auth Method can be created with [`-token-locality=global`](/commands/acl/auth-method/create#token-locality). All tokens with global locality will be replicated to all secondary datacenters.
+
+Either a global token, or a local token in the target datacenter, is required to perform a cross-datacenter request.
```

Comment:
```suggestion
Either a global token, or a local token from the target datacenter, is required to perform a cross-datacenter request.
```

I'm not sure what others thing, but to me "from" reads a bit better here than "in".